011-quarkus-panache-rest-flyway/src/main/java/io/quarkus/qe/ServiceResource.java

```diff
@@ -0,0 +1,6 @@
+package io.quarkus.qe;
+
+import io.quarkus.hibernate.orm.rest.data.panache.PanacheEntityResource;
+
+public interface ServiceResource extends PanacheEntityResource<ServiceEntity, Long> {
```

Comment:
This is another workaround as I could not add the application-service relationship using the application REST endpoint... @rsvoboda do you think this is worthy to be reported as well? I would say this is a very common functionality that many users will eventually need and try.

Reply:
I've deleted the Service resource and reported the issue https://github.com/quarkusio/quarkus/issues/13842